py3-cython.yaml

```diff
@@ -40,17 +40,42 @@ subpackages:
       provider-priority: ${{range.value}}
       provides:
         - ${{package.name}}
-        - cython
     pipeline:
       - uses: py/pip-build-install
         with:
           python: python${{range.key}}
       - uses: strip
 
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}-bin
+    description: Executable binaries for ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-${{vars.pypi-package}}
+      provides:
+        - cython
+      provider-priority: ${{range.value}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr
+          mv ${{targets.contextdir}}/../py${{range.key}}-${{vars.pypi-package}}/usr/bin ${{targets.contextdir}}/usr
```

Comment:
This is kind of yucky.  Depends on a specific value of targets.subpkgdir for another package.
another option would be for the non -bin version to 

```
  mkdir -p ./cleanup/${{range.key}/cython
  mv ${{targets.contextdir}}/usr/bin/cython ./cleanup/${{range.key}/
```

And then here you would:

```
  mkdir -p ${{targets.contextdir}}/usr/bin
  mv ./cleanup/${{range.key}}/cython ${{targets.contextdir}}/usr/bin
```